moebot_bot/bot/commands/irl.go

```diff
@@ -0,0 +1,34 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/camd67/moebot/moebot_bot/bot/permissions"
+	"github.com/camd67/moebot/moebot_bot/util/db"
+	"github.com/camd67/moebot/moebot_bot/util/reddit"
+)
+
+type IrlCommand struct {
+	PermChecker  permissions.PermissionChecker
```

Comment:
Is there a reason we're including perm checker in this? Doesn't look like we're passing it in when we make the command nor use it.